akka-http-core/src/main/scala/akka/http/model/parser/HeaderParser.scala

```diff
@@ -99,7 +99,7 @@ object HeaderParser {
     "range",
     "proxy-authenticate",
     "proxy-authorization",
-    "range",
```

Comment:
It's a duplication from 3 lines above. Maybe delete the other one to keep the sorting?